docs/user/supported_devices.rst

```diff
@@ -427,7 +427,9 @@ ramips-mt7621
 
 * ZyXEL
 
+  - Multy M1
```

Comment:
Remove this line please, Gluon cares for the model number, not the marketing term.